src/app/registration/contact-form/contact-form.component.ts

```diff
@@ -116,6 +134,20 @@ export class ContactFormComponent implements OnInit {
     return this.myForm.get('enablePhone');
   }
 
+  updateMaxLength(placeholder: string): void {
+    const placeholderWithoutDialCode = placeholder.slice(this.dialCode.length + 1);
+    this.maxLength = placeholderWithoutDialCode.length;
+    console.log("MAX LENGTH: ", this.maxLength);
+  }
+  
+  onCountryChange(event: any): void{
+    console.log("event: ", event);
```

Comment:
Remove console logs (typ)

Reply:
Removed this!

Addressed in: https://github.com/bcgov/parks-reso-public/pull/304/commits/1a599013cd01abb93ec6705be0b8577b355eb442